README.md

```diff
@@ -1 +1,92 @@
 # scrooge-v2
+## Endpoint
+### POST /events/_create
+request
+```
+{
+    "name": "Koiki Camp",    ... (required)
+    "transferCurrency": "JPY"        ... (required)
+}
+```
+
+response (201 status)
+```
+{
+    "name": "Koiki Camp",
+    "id": "5a226c2d7c245e14f33fc5a8",
+    "createdAt": "2017-12-02T16:52:45.52",
+    "updatedAt": "2017-12-02T16:52:45.52",
+    "transferCurrency": "JPY",
+    "groups": [{ "scrooges": [], "members": []}],
+    "transferAmounts": []
+}
+```
+
+### POST /events/{eventId}/groups/_add
+request
+```
+{
+    "name": "Koiki Camp",    ... (required)
+    "transferCurrency": "JPY"        ... (required)
+}
+```
+
+### POST /events/{eventId}/groups/{groupId}/memberNames/{memberName}
+WIP
+
+### POST /events/{eventId}/groups/{groupId}/scrooges/{scroogeId}
+```
+{
+    "memberName": "Nabnab", ... (required)
+    "paidAmount": 200,      ... (required)
+    "currency": "JPY",      ... (required)
+    "forWhat": "rent-a-car" ... (optional)
+}
+```
+
+### GET /events/{eventId}
```

Comment:
Is this HTTP GET end point or message from wss?

Reply:
This is a schema of HTTP GET endpoint. I did not design wss schema yet.